src/js/components/molecules/article-table/article-table.ts

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { ArticleTableProps } from './article-table.models';
+import WithRender from './article-table.vue.html';
+
+@WithRender
+@Component({
+    name: 'ArticleTable',
+})
+export class ArticleTable extends Vue implements ArticleTableProps {
+    @Prop({ type: String, required: true })
```

Comment:
🐜 prefer to see the prop statement on one line if possible

Reply:
I think prettier put it on 2…

Reply:
it should use 120 characters per line, but okay.